src/components/Question.jsx

```diff
@@ -107,28 +120,42 @@ export default class Question extends React.Component {
 
   render() {
     const question = this.props.question;
+    const subQuestion = question[Constants.HAS_SUBQUESTION];
     const options = this.context.options;
-
+    const renderQuestion = this.renderQuestion(question);
     if (FormUtils.isHidden(question)) {
       return null;
     }
+    if (!FormUtils.isRelevant(question) &&
+        (this.context.options.debugMode || FormUtils.isQuestionMatchingStartingQuestionId(subQuestion, options.startingQuestionId))) {
+      return (
+          <div className="show-irrelevant">
+            {renderQuestion}
+          </div>
+      );
+    }
     if (!FormUtils.isRelevant(question)) {
       return null;
     }
+
+    return renderQuestion;
+  }
+
+  renderQuestion(question) {
     if (FormUtils.isAnswerable(question) && !FormUtils.isSection(question)) {
       if (PRETTY_ANSWERABLE_LAYOUT) {
         return (
-          <div id={question['@id']}>
-            <div className="panel-title answerable-question">{this.renderAnswers()}</div>
-            <div className="answerable-subquestions">{this.renderSubQuestions()}</div>
-          </div>
+            <div id={question['@id']}>
```

Comment:
uff there are a lot of formating here ... IMO, next time its good practice not to commit it together (hope it won't be too messy merging it with pretifier

Reply:
Agree, hopefully Prettier will help with this